src/openrct2/cheats.c

```diff
@@ -420,6 +422,58 @@ static void cheat_set_staff_speed(uint8 value)
 	}
 }
 
+static void cheat_own_all_land() {
+	uint8 prev_sandbox, prev_no_money, i;
+	uint16 spriteIndex, nextSpriteIndex, spawns_x[2];
+	int min, max, flags, cmd, type;
+	rct_peep *peep;
+
+	//Store states of sandbox, no money, and spawn points to restore when finished
+	prev_sandbox = gCheatsSandboxMode;
+	prev_no_money = gParkFlags & PARK_FLAGS_NO_MONEY;
+	for (i = 0; i < 2; i++) {
```

Comment:
There isn't one for this yet. Once this is merged, I'll likely create a pr for RCT_PARK_ENTRANCES and RCT_PEEP_SPAWNS macros. But I'd prefer to do that another time, since if i did it in here i'd need to update all the existing 2's throughout the code and can overcomplicate this.